### PR TITLE
Bug: Departure City now being rendered and added to the Trips table in the database

### PR DESCRIPTION
**Pull request by jldaggs88**

### CODE REVIEW

File: server/weather.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable function-paren-newline */
```
Comment (jstrzesz) [inline server/weather.js:1]: do not use eslint-disable

File: server/weather.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+/* eslint-disable implicit-arrow-linebreak */
```
Comment (jstrzesz) [inline server/weather.js:2]: do not use eslint-disable

File: client/components/Itinerary.js [hunk 3]
```diff
@@ -74,6 +75,7 @@ const Itinerary = ({ currentUser, currentTrip, day }) => {
                 day,
               })
               .then((response) => {
+                // console.log('response from db for activity list', response);
```
Comment (jstrzesz) [inline client/components/Itinerary.js:80]: remove

File: client/components/preferences.js [hunk 4]
```diff
@@ -72,13 +75,19 @@ const ContinuousSlider = ({ currentUser, allOtherUsers, setClickedPage }) => {
     setEndDate(event);
   };
 
+  const handleDepartureInput = (event) => {
+    console.info(event);
```
Comment (jstrzesz) [inline client/components/preferences.js]: remove

File: server/queries.js [hunk 5]
```diff
@@ -163,6 +163,7 @@ const grabPlaces = async (req, res) => {
 
 // add planned trip
 const planTrip = async (req, res) => {
+  console.info(req, 'in queeries');
```
Comment (jstrzesz) [inline server/queries.js:166]: remove

File: client/components/preferences.js [hunk 6]
```diff
@@ -1,6 +1,8 @@
+/* eslint-disable max-len */
```
Comment (jstrzesz) [inline client/components/preferences.js:1]: do not use eslint-disable

File: client/components/preferences.js [hunk 7]
```diff
@@ -136,7 +137,7 @@ const ContinuousSlider = ({ currentUser, allOtherUsers, setClickedPage }) => {
     <Container fixed classes={{ root: 'preferences-container' }}>
       <div className="text-inputs">
         <Typography variant="h2">Plan a Trip</Typography>
-        <label htmlFor="text">
+        {/* <label htmlFor="text">
```
Comment (jstrzesz) [inline client/components/preferences.js:140]: remove if this is old code

**Review by gjones2019 — DISMISSED** [review]

**Comment by jldaggs88** [comment]
Changes made per Josh's request.

**Review by joshjnunez — APPROVED** [review]

**Review by gjones2019 — APPROVED** [review]

**Pull request merged**